include/pros/apix.h

```diff
@@ -369,6 +369,7 @@ void queue_reset(queue_t queue);
 typedef enum v5_device_e {
 	E_DEVICE_NONE = 0,
 	E_DEVICE_MOTOR = 2,
+  E_DEVICE_IMU = 6,
```

Comment:
nit: looks like you need a run of clang format

Reply:
ah, it didn't catch this one because it wasn't tracked

Reply:
A couple of imu.hpp's comments are also messed up.